presto-product-tests/src/main/java/com/facebook/presto/tests/sqlserver/Select.java

```diff
@@ -141,4 +157,21 @@ public void testAllDatatypes()
                         row(null, null, null, null, null, null, null, null, null, null, null, null, null, null)
                 );
     }
+
+    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
+    public void testCreateTableAsSelect()
+            throws SQLException
+    {
+        String sql = format(
+                "CREATE TABLE %s AS SELECT * FROM %s", CREATE_TABLE_AS_SELECT, NATION_TABLE_NAME);
+        onPresto().executeQuery(sql);
+
+        sql = format(
```

Comment:
I believe you should not use Presto to check results.
It's like using a test subject for the test itself.

How hard would it be to introduce `onSqlServer().executeQuery(...)`?